src/query/ee/src/storages/fuse/operations/ngram_index.rs

```diff
@@ -395,10 +394,21 @@ impl AsyncTransform for NgramIndexTransform {
             let state =
                 BloomIndexState::from_bloom_index(&index, index_location, new_ngram_index_column)?;
 
-            // remove old bloom index meta
+            // remove old bloom index meta and filter
             if let Some(cache) = CacheManager::instance().get_bloom_index_meta_cache() {
                 cache.evict(&index_path);
             }
+            if let Some(cache) = CacheManager::instance().get_bloom_index_filter_cache() {
+                let cache_key = format!(
```

Comment:
the cache key is the new created ngram bloom, but we need to evict the old cache key.